maps/donut3.dmm

```diff
@@ -11836,6 +11836,7 @@
 /obj/decal/tile_edge/stripe/corner{
 	dir = 6
 	},
+/obj/mapping_helper/access/hos,
```

Comment:
```suggestion
/obj/mapping_helper/access/security,
```
Interior armoury turrets don't open fire upon anyone who has Security access. This is how it was prior to your armoury remap.